src/models/Article.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const articleSchema = new mongoose.Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+    },
+    author: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+
+    urlToImage: {
+      type: String,
+      required: true,
+    },
+    violence_type: {
+      type: String,
+      required: true,
```

Comment:
do u use the type violence in FE? it has to be array in the future

Reply:
I changed it to array